x-pack/test/security_solution_cypress/cypress/tasks/api_calls/cases.ts

```diff
@@ -28,3 +30,23 @@ export const createCase = (newCase: TestCase) =>
     },
     headers: { 'kbn-xsrf': 'cypress-creds', 'x-elastic-internal-origin': 'security-solution' },
   });
+
+export const getCases = () =>
+  rootRequest<CasesFindResponse>({
+    method: 'GET',
+    url: 'api/cases/_find',
+  });
+
+export const deleteCases = async () => {
+  getCases().then(($response) => {
+    if ($response.body.cases.length > 0) {
+      const ids = $response.body.cases.map((myCase) => {
+        return `"${myCase.id}"`;
+      });
+      rootRequest({
+        method: 'DELETE',
+        url: `/api/cases?ids=[${ids}]`,
+      });
+    }
+  });
+};
```

Comment:
Since we are in an _async_ function, it would make more sense to use `await` instead of `then`:

```suggestion
export const deleteCases = async (): Promise<void> => {
  const casesResponse = await getCases();
  const ids = casesResponse.body.cases.map(({id}) => `"${id}"`);
  if (ids.length === 0) {
    return;
  }
  await rootRequest({
    method: 'DELETE',
    url: `/api/cases?ids=[${ids}]`,
  });
};
```